atox/src/main/kotlin/ui/Util.kt

```diff
@@ -38,3 +46,24 @@ internal data class Px(val px: Float) : Size {
 internal data class Dp(val dp: Float) : Size {
     override fun asPx(res: Resources): Px = Px(dpToPx(dp, res))
 }
+
+/*
+ * Function will save the image with the input bitmap and name to the cache directory as a png format.
+ * Then it will return the Uri of the image.
+ */
+internal fun saveImageForSharing(context: Context, image: Bitmap, name: String): Uri? {
```

Comment:
I think this function should be a private one in the file it's used until we actually need it elsewhere since it's pretty specific.